Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MarkdownTextBlock/MarkdownTextBlockPage.xaml.cs

```diff
@@ -79,5 +84,15 @@ private void SetInitalText(string text)
                 await Launcher.LaunchUriAsync(new Uri(e.Link));
             }
         }
+
+        // Custom Code Block Renderer
+        private void MarkdownText_CodeBlockResolving(object sender, CodeBlockResolvingEventArgs e)
```

Comment:
This is basically a parser event (I can't see where it's raised from yet).  Should it be `ParseCustomCodeBlock`?

I didn't see the term `Resolving` used elsewhere in the parser, so just want to make sure we're consistent and not introducing a bunch of terms.

Reply:
It is used by `ImageResolving` in MarkdownTextBlock.Events.cs.

Reply:
Commented on it down there when I found it, this is fine. :)